app/assets/javascripts/admin/confirm.js

```diff
@@ -0,0 +1,9 @@
+$.fn.twitter_bootstrap_confirmbox.defaults = {
```

Comment:
This can be a CoffeScript file, but had some issues getting this to work as CoffeScript instead of regular JS.